src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,125 @@
+package duke.tasklist;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import duke.task.Task;
+
+/**
+ * Keeps and handles all operations regarding the user tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
```

Comment:
There should be an extra line space to keep consistency

Reply:
Fixed. Thanks